### ISSUE TITLE
Add RPC Blockchain support

### ISSUE THREAD

**Issue by notmandatory** [bitcoindevkit/bdk-ffi#117]

**Comment by notmandatory** [comment]
@ConorOkus do you have a particular project you want to use this with? I'm only planning to manually test it for now in the `bdk-kotlin` project. If you have a project to test it in that will give me another way to verify it does what you need.

**Comment by ConorOkus** [comment]
Yeah, it's early stages but I've started building an Android wallet using the Kotlin bindings from LDK [here](https://github.com/ConorOkus/uMlando-wallet)

I'd like to use it to programmatically get stuff like the latest block hash in the local regtest environment e.g instead of using the Esplora API [here](https://github.com/ConorOkus/uMlando-wallet/blob/master/app/src/main/java/com/example/umlandowallet/data/remote/ServiceImpl.kt#L12)

**Comment by ConorOkus** [comment]
Yes absolutely, I was planning to run the node locally and use a virtual device simulator for what it's worth.

**Comment by notmandatory** [comment]
If you want to try something like [`nigiri`](https://github.com/vulpemventures/nigiri) it can spin up a local regtest bitcoind with an electrum or esplora server.
